### ISSUE TITLE
Rebase on Lobsters?

### ISSUE THREAD

**Issue by callahad** [ozten/commit#1]
Seems like the lamernews codebase is unmaintained (the last commit was a year ago, today). I wonder if https://github.com/jcs/lobsters, which powers https://lobste.rs/ would be a more suitable, maintained base for commit.ws?

It might also be overkill and a rabbit hole, but the complexity of switching only grows as the site accumulates real content, so Commit is probably at the least-painful moment to do this.

**Comment by ozten** [comment]
Yes, lamernews is just a MVP to see if there is a viable community.

Thanks for suggesting lobste.rs, I hadn't thought of that!

My rough plan was to do a serverless architecture, but nothing is set in stone.
